src/Http/Controllers/DispatchJobController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Llaski\NovaScheduledJobs\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Llaski\NovaScheduledJobs\Rules\JobExist;
+
+class DispatchJobController
+{    
+    /**
+     * Dispatch job command.
+     *
+     * @param  \Illuminate\Http\Request $request
+     */
+    public function create(Request $request)
+    {
+        $data = $request->validate([
+            'command' => ['required','string', new JobExist()]
+        ]);
+    
+        dispatch(new $data['command']());
```

Comment:
@tanjemark The command should be resolved from the IOC container as there could be dependecies that need to be injected before being dispatched.

Reply:
Hey @tanjemark just wanted to note this comment again in case ya missed it initially. Once this is fixed I will merge it in.